src/pages/Explore/Explore.css

```diff
@@ -0,0 +1,110 @@
+.explore-page-container
+{
+    padding: 2rem 6rem;
+    min-height: calc(100vh - 107.23px);
+    grid-column: 2/3;
+    position: relative;
+    display: grid;
+    grid-template-columns: minmax(60rem, 1fr) 35rem;
+    gap: 2rem;
+    background-color: var(--background-color);
+}
+
+.explore-feed-container
+{
+    min-height: 100%;
+    display: flex;
+    padding: 1rem 0;
+    flex-direction: column;
+    gap: 1rem;
+    justify-content: flex-start;
+    align-items: center;
+}
+
+.user-post
+{
+    height: 50rem;
+    width: 60rem;
+    /* width: 35rem; */
+    border-radius: 8px;
+    background-color: white;
+}
+
+.explore-suggestion-container
+{
+    min-height: fit-content;
+    padding: 1rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+}
+
+.whats-happening-container
+{
+    width: 100%;
+    height: fit-content;
+    padding:  1rem 2rem;
+    border-radius: 8px;
+    text-align: left;
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+}
+
+.explore-suggestion-container hr
+{
+    width: 100%;
+}
+
+.active-contacts-container
+{
+    width: 100%;
+    height: fit-content;
+    padding:  1rem 2rem;
+    border-radius: 8px;
+    text-align: left;
+    background-color: white;
+}
+
+@media only screen and (max-width:1335px){
+    .explore-page-container
+    {
+        grid-template-columns: 1fr;
+    }
+
+    .explore-suggestion-container
+    {
+        display: none;
+    }
+}
+
+@media only screen and (max-width:1015px){
+    .explore-page-container
+    {
+        min-height: calc(100vh - 157.23px);
+    }
+}
+
+@media only screen and (max-width: 970px)
+{
+    .user-post
+    {
+        width: 50rem;
+    }
+}
+
+@media only screen and (max-width: 870px)
+{
+    .user-post
+    {
+        width: 40rem;
+    }
+}
+
+@media only screen and (max-width: 770px)
```

Comment:
try not to use pixels for media queries.

Reply:
Yeah, after your suggestion I read more about it and you are right.
Updated all my media queries to ems